cmd/install/install.go

```diff
@@ -85,6 +86,7 @@ func NewInstallCmd() *cobra.Command {
 	cmd.AddCommand(NewInstallStorageCmd())
 	cmd.AddCommand(NewInstallAliasCmds(storageApps, "storage-")...)
 	cmd.AddCommand(velero.NewApp().NewInstallCmd())
+	cmd.AddCommand(consul.NewApp().NewInstallCmd())
```

Comment:
Would you mind putting this right after coredumphandler to keep it alphabetical?

Reply:
Fixed ordering